src/main/webapp/lectures/lectures.js

```diff
@@ -20,9 +20,12 @@ const REDIRECT_PAGE = '/view/';
 
 loadLectureList();
 
-/** Fetches data from servlet and sets it in the lecture selection page. */
+/**
+ * Fetches avaiable Lectures from LectureListServlet and sets them in the
+ * lecture selection page.
+ */
 async function loadLectureList() {
-  const response = await fetch('/lecture');
+  const response = await fetch('/lecture-list');
```

Comment:
can we turn this into `ENDPOINT_LECTURE_LIST`, and include the constant in the function comment?